test/e2e/tests/privacy/onboarding-privacy.spec.js

```diff
@@ -0,0 +1,283 @@
+const { strict: assert } = require('assert');
+const {
+  TEST_SEED_PHRASE,
+  withFixtures,
+  importSRPOnboardingFlow,
+  defaultGanacheOptions,
+  onboardingBeginCreateNewWallet,
+  onboardingChooseMetametricsOption,
+  onboardingCreatePassword,
+  onboardingRevealAndConfirmSRP,
+  onboardingCompleteWalletCreation,
+  regularDelayMs,
+} = require('../../helpers');
+const FixtureBuilder = require('../../fixture-builder');
+
+describe('MetaMask onboarding @no-mmi', function () {
+  it("doesn't make any network requests to infura before onboarding is completed", async function () {
```

Comment:
This is a test file splited from the original `test/e2e/tests/onboarding/onboarding.spec.js` because these tests are more privacy-related and should be categorized in the privacy folder. I will create a follow-up PR to convert this test file to POM and TypeScript to avoid making the PR too large.